movies_test.py

```diff
@@ -0,0 +1,134 @@
+"""Tests the functions of the code.py file"""
+
+import movies
+
+def customer_db():
+    """Define Customer database for testing"""
+    return {
+        "Bob" : {"movies":[],
+                 "cash" : 100.0},
+        "Jim" : {"movies": ["Xmen 8: The Xmennening"],
+                 "cash" : 10.0},
+        "Cary" : {"movies": ["Gigli: The Play: The Book: The movie", "The Bromance"],
+                  "cash" : 120.0},
+        "Ricci": {"movies": [],
+                  "cash" : 4.0}
+    }
+
+def movie_db():
+    """Define movie database for testing"""
+    return {
+        "Xmen" : {"title" : "Xmen 8: The Xmennening", "seats_avaliable" : 10},
+        "Bromance" : {"title" : "The Bromance", "seats_avaliable" : 20},
+        "Gigli" : {"title" : "Gigli: The Play: The Book: The movie", "seats_avaliable" : 102}
+    }
+
+def test_enough_money():
+    """Tests the enough money function"""
+    customers = customer_db()
+    bob = customers["Bob"]
+    ricci = customers["Ricci"]
+    assert not movies.enough_money(ricci)
+    assert movies.enough_money(bob)
+
+def test_charge():
+    """Tests charge function"""
+    customers = customer_db()
+    bob = customers["Bob"]
+    bobs_actual_cash = movies.charge(bob)["cash"]
+    bobs_expected_cash = 95.0
+    assert bobs_actual_cash == bobs_expected_cash
+    jim = customers["Jim"]
+    jims_actual_cash = movies.charge(jim)["cash"]
+    jims_expected_cash = 5.0
+    assert jims_actual_cash == jims_expected_cash
+
+def test_dispense_ticket():
+    """Test adding ticket to customers account"""
+    customers = customer_db()
+    movies_ = movie_db()
+    bob = customers["Bob"]
+    xmen = movies_["Xmen"] #here
+    bobs_library = movies.dispense_ticket(xmen, bob)["movies"]
+    bobs_expected_library = ["Xmen 8: The Xmennening"]
+    assert bobs_library == bobs_expected_library
+    jim = customers["Jim"]
+    bromance = movies_["Bromance"]
+    jims_library = movies.dispense_ticket(bromance, jim)["movies"]
+    jims_expected_library = ["Xmen 8: The Xmennening", "The Bromance"]
+    assert jims_library == jims_expected_library
+
+def test_remove_seat():
```

Comment:
So what I was getting at with this one, is that you should have a test in here similar to what you did here:
``` 
+    ricci = customers["Ricci"]
 +    assert not movies.enough_money(ricci)
```
create a move that has zero seats and then try to remove a seat from it